pulsar-function-go/conf/conf.go

```diff
@@ -87,17 +87,23 @@ func (c *Conf) GetConf() *Conf {
 		flag.Usage()
 	}
 
+	if confContent == "" && confFilePath == "" {
+		log.Errorf("no yaml file or conf content provided")
+		return nil
+	}
+
 	if confFilePath != "" {
 		yamlFile, err := ioutil.ReadFile(confFilePath)
-		if err != nil {
+		if err == nil {
+			err = yaml.Unmarshal(yamlFile, c)
+			if err != nil {
+				log.Errorf("unmarshal yaml file error:%s", err.Error())
+				return nil
+			}
+		} else if (err != nil || os.IsNotExist(err)) && confContent == "" {
```

Comment:
err != nil && !os.IsNotExist(err)

Reply:
@sijie then we also need to handle `err != nil && os.IsNotExist(err) && confContent == ""`, right?

Reply:
yes. you need to handle two case:

if the err is not IsNotExist, it means it encounters other issues on reading the configuration, it should panic.
if the err is IsNotExist, then you check if confContent == "", if it doesn't exist, you panic.